src/main/java/Database.java

```diff
@@ -0,0 +1,75 @@
+import java.util.ArrayList;
+
+public class Database {
+    public static ArrayList<Task> taskList = new ArrayList<>();
+
+    public static void markDone(String arg) {
+        try{
+            int index = Integer.parseInt(arg);
+            taskList.get(index - 1).markAsDone();
+            markDoneResponse(index);
+        } catch(NumberFormatException e) {
+            System.out.println("The command is not recognised.");
+        } catch (IndexOutOfBoundsException e){
+            System.out.println("The index is out of range. This action is aborted.");
+        }
+    }
+
+    public static void listAll(){
+        int i = 1;
+        for(Task task: taskList){
+
```

Comment:
Perhaps you can delete the blank line here

Reply:
Yes thanks. My terrible coding habit of adding blank lines mindlessly